File: Java/769. Max Chunks To Make Sorted.java
class Solution {
    public int maxChunksToSorted(int[] arr) {
        int res=0;
        int m=0;
        for(int i=0; i<arr.length; i++){
            m=Math.max(m,arr[i]);
            if(m==i){
                res++;
            }
        }
        return res;
    }
}

File: C#/769. Max Chunks To Make Sorted.cs
public class Solution {
    public int MaxChunksToSorted(int[] arr) {
        int res=0;
        int m=0;
        for(int i=0; i<arr.Length; i++){
            m=Math.Max(m,arr[i]);
            if(m==i){
                res++;
            }
        }
        return res;
    }
}
